index.html

```diff
@@ -0,0 +1,715 @@
+                opacity: 0.0;
+                z-index: initial;
+            }
+
+            .ShowOutput1 {
+                animation: FadeInO1 .75s ease-in forwards;
+                -webkit-animation: FadeInO12 .75s ease-in forwards;
+                display: inline;
+                height: 100%;
+                width: 100%;
+                position: absolute;
+                z-index: 1;
+            }
+
+            @keyframes FadeInO1 {
+                to {
+                    opacity: 1.0;
+                }
+            }
+
+            @-webkit-keyframes FadeInO12 {
+                to {
+                    opacity: 1.0;
+                }
+            }
+
+            .HideOutput1 {
+                animation: FadeOut1 .75s ease-in forwards;
+                -webkit-animation: FadeOut12 .75s ease-in forwards;
+                display: inline;
+                height: 100%;
+                width: 100%;
+                position: absolute;
+                z-index: 2;
+                opacity: 1.0;
+            }
+
+            @keyframes FadeOut1 {
+                to {
+                    opacity: 0.0;
+                }
+            }
+
+            @-webkit-keyframes FadeOut12 {
+                to {
+                    opacity: 0.0;
+                }
+            }
+
+            .HideOutput2 {
+                animation: FadeOut .75s ease-in forwards;
+                -webkit-animation: FadeOut2 .75s ease-in forwards;
+                display: inline;
+                height: 100%;
+                width: 100%;
+                position: absolute;
+                z-index: 2;
+                opacity: 1.0;
+            }
+
+            @keyframes FadeOut {
+                to {
+                    opacity: 0.0;
+                }
+            }
+
+            @-webkit-keyframes FadeOut2 {
+                to {
+                    opacity: 0.0;
+                }
+            }
+
+            .ShowOutput2 {
+                animation: FadeIn .75s ease-in forwards;
+                -webkit-animation: FadeIn2 .75s ease-in forwards;
+                display: inline;
+                height: 100%;
+                width: 100%;
+                position: absolute;
+                z-index: 2;
+            }
+
+            @keyframes FadeIn {
+                to {
+                    opacity: 1.0;
+                }
+            }
+
+            @-webkit-keyframes FadeIn2 {
+                to {
+                    opacity: 1.0;
+                }
+            }
+        </style>
+  <iframe id="Layer1" class="Off"></iframe>
+  <iframe id="Layer2" class="Off"></iframe>
+  <script>
+            function Start() {
+                window.WPSS = {};
+                window.WPSS.Duration = 45000;
+                window.WPSS.CurrentOutput = "Layer1";
+                window.WPSS.playlist = document.querySelectorAll("div#Playlist a");
+                window.WPSS.playlist.RandomLength = window.window.WPSS.playlist.length;

```

Comment:
Why do you add a new storage for this value, why don't you use window.window.WPSS.playlist.length in the code